include/modem/lte_lc.h

```diff
@@ -234,6 +240,37 @@ int lte_lc_edrx_param_set(const char *edrx);
  */
 int lte_lc_edrx_req(bool enable);
 
+/**
+ * @brief      Set the parameters for the default PDP context.
+ * Must be called prior to `lte_lc_init` or `lte_lc_init_and_connect`.
+ *
+ * @param[in]  type            PDP type
+ * @param[in]  apn             APN Name
+ * @param[in]  ip4_addr_alloc  0 - Allocate IPV4 address via NAS signaling
+ *	(default), 1 - Allocate IPV4 address via DHCP.
+ * @param[in]  nslpi           0 - NSLPI value from configuration is used
+ *	(default), 1 - Value "Not configured".
+ * @param[in]  securePCO       0 - Protected PCO transmission is not requested
+ *	(default), 1 - Protected PCO transmission is requested
+ *
+ * @return     Zero on success or (negative) error code otherwise.
+ */
+int lte_lc_pdp_context_set(enum lte_pdp_type type, const char *apn,
```

Comment:
Would it make sense to create a struct with PDP context config parameters and pass a pointer to it as an argument?
```
struct pdp_context_param {
	char *apn;
	bool ip4_addr_alloc;
etc...
};
```
It should make the configuration easily extensible if needed (I still see a lot of hardcoded `0` in the command).

Reply:
The `0` are all reserved values and should be `0` per AT commands doc.

Reply:
I see, in that case, I'm not pushing for this change, just fix the alignment of the arguments.